packages/dds/sequence/src/sequence.ts

```diff
@@ -597,6 +597,10 @@ export abstract class SharedSegmentSequence<T extends ISegment>
         this.loadFinished();
     }
 
+    protected applyStashedOp(content: any) {
+        return this.client.applyStashedOp(content);
```

Comment:
within sequence there is another dds like structure for shared intervals. look at the resubmit code. I don't think we should add handling for it in this PR, but we should fail if we see a non-merge tree op. you could handle that here, or fail in the default case of applyStashedOp in client.ts